### PR TITLE
Skip generic shows from duplicate removal actions

### LINKED ISSUE #5856: Scheduled series recording skipping over episodes when Season/Episode data unavailable in Guide
**Describe the bug**
When scheduling a series recording in Jellyfin, there is erratic behavior where individual episodes in the series will (inaccurately) be left off of the recording schedule. I've noticed this only for series where there is no Season or Episode information in the EPG Guide (i.e. local news). Even though I have the options to record every episode, and not to check against episodes already in the library, clicking around in the guide and checking the scheduled recordings will somehow randomly change the episodes that are scheduled to be recorded. I am using XML guide data and have observed this behavior without refreshing the guide data. Screenshots below.

**System (please complete the following information):**
 - OS: Windows 10
 - Virtualization: N/A
 - Clients: N/A
 - Browser: Chrome 89, Edge 89
 - Jellyfin Version: 10.7.2
 - Playback: N/A 
 - Installed Plugins: none
 - Reverse Proxy: none
 - Base URL: none
 - Networking: Host
 - Storage: local

**To Reproduce**

1. Open Guide

2. Click on a episode that is part of a series

3. Schedule recording of series (see screenshot 1, notice both series and episode icons are red)
Scheduled Recordings 1

4. Click on series settings, configure to record all episodes and deselect option to compare against episodes already in library (see screenshot 2, notice many episodes are detected but only the first is scheduled to record)
Scheduled Recordings 2

5. Go to back to guide and select same episode that was selected in Step 2 (see screenshot 3, notice episode is no longer scheduled to record)
Scheduled Recordings 3

6. Click on series settings (see screenshot 4, notice original episode is no longer scheduled to record)
Scheduled Recordings 4

**Expected behavior**
Expect all episodes to be scheduled to record (in the screenshots, all nightly episodes are "new", so they all should have been scheduled to begin with.
Do not expect individual episodes to be un-scheduled simply by clicking back and forth with the guide and scheduled recordings, without making any changes

**Screenshots**
Screenshots attached here

**Additional context**
I am using an OTA antenna connected to an HDHomerun 4k
Guide data is imported from XML file generated by zap2xml

### PR DESCRIPTION
**Pull request by abpei**
**Changes**
Workaround for #5856.  Some shows (news, sports, etc.) get published in guides without subkey and are therefore hit by duplicate recording removal, even though the <new> tag is present.  This skips removal of duplicate shows that do not have SubKeys.

**Issues**
Fixes #5856

### CODE REVIEW

File: Emby.Server.Implementations/LiveTv/EmbyTV/EmbyTV.cs [hunk 1]
```diff
@@ -2221,6 +2221,13 @@ private void SearchForDuplicateShowIds(List<TimerInfo> timers)
                 {
                     continue;
                 }
+                
+                //Skip ShowId without SubKey from duplicate removal actions - https://github.com/jellyfin/jellyfin/issues/5856
+	            string ShowIdSubKey = group.Key.Substring(group.Key.Length - 4);
+				if (ShowIdSubKey == "0000" )
```
Comment (cvium) [inline Emby.Server.Implementations/LiveTv/EmbyTV/EmbyTV.cs:2227]:
```suggestion
                // Skip ShowId without SubKey from duplicate removal actions - https://github.com/jellyfin/jellyfin/issues/5856
				if (group.Key.EndsWith("0000")) 
```

**Review by cvium — APPROVED** [review]

File: Emby.Server.Implementations/LiveTv/EmbyTV/EmbyTV.cs [hunk 2]
```diff
@@ -2221,6 +2221,12 @@ private void SearchForDuplicateShowIds(List<TimerInfo> timers)
                 {
                     continue;
                 }
+                
+                // Skip ShowId without SubKey from duplicate removal actions - https://github.com/jellyfin/jellyfin/issues/5856
+				if (group.Key.EndsWith("0000")) 
+				{
+					continue;
+				}
```
Comment (Bond-009) [inline Emby.Server.Implementations/LiveTv/EmbyTV/EmbyTV.cs:2229]: Indented with tabs instead of spaces

**Review by Bond-009 — CHANGES_REQUESTED** [review]

**Review by crobibero — APPROVED** [review]

**Review by Bond-009 — CHANGES_REQUESTED** [review]
It's still indented with tabs instead of spaces

**Review by abpei — COMMENTED** [review]
I hope it took the update this time. Sorry, I don't do this every day.

**Review by Bond-009 — APPROVED** [review]

**Pull request merged**